test/Microsoft.Azure.SignalR.Tests/E2ETest/TestServer.cs

```diff
@@ -30,7 +32,16 @@ protected override Task StartCoreAsync(string serverUrl, ILoggerFactory loggerFa
 
         public override async Task StopAsync()
         {
+
             await _host.StopAsync();
+
+            // dispose client connections
+            var clientConnections = _host.Services.GetRequiredService<IClientConnectionManager>().ClientConnections;
+            clientConnections.Clear();
```

Comment:
>clientConnections [](start = 12, length = 17)

do we need to dispose the connections before clear the set?

Reply:
These connections are `ServiceConnectionContext`, which don't implement `IDisposable`, so we don't need to dispose them.